cli/azd/pkg/auth/manager.go

```diff
@@ -1017,16 +1104,20 @@ type persistedSecret struct {
 
 // federated auth token providers
 var (
-	gitHubFederatedAuth federatedTokenProvider = "github"
+	GitHubFederatedTokenProvider         FederatedTokenProvider = "github"
+	AzurePipelinesFederatedTokenProvider FederatedTokenProvider = "azure-pipelines"
 )
 
 // token provider for federated auth
-type federatedTokenProvider string
+type FederatedTokenProvider string
```

Comment:
Why did it become exported?

Reply:
I'll undo this - I think I had this exported earlier iteration when it was a parameter to a `Login` method.